src/test/java/picard/analysis/CollectAlignmentSummaryMetricsTest.java

```diff
@@ -51,8 +55,7 @@ public String getCommandLineProgramName() {
     public void test() throws IOException {
         final File input = new File(TEST_DATA_DIR, "summary_alignment_stats_test.sam");
         final File reference = new File(TEST_DATA_DIR, "summary_alignment_stats_test.fasta");
-        final File outfile = File.createTempFile("alignmentMetrics", ".txt");
-        outfile.deleteOnExit();
+        final File outfile = File.createTempFile("test", ".txt", TEMP_OUTPUT_DIR);
```

Comment:
It looks like this is a common case. You could add `createTempFile` to `CommandLineProgramTest` to avoid some typing.

Reply:
not much less typing...but OK.